app/javascript/app/components/nav/nav.js

```diff
@@ -2,8 +2,12 @@ import { connect } from 'react-redux';
 
 import Component from './nav-component';
 
+// const actions = null;
```

Comment:
Should be the `actions = null` uncommented until the `setup` flow is implemented?